setup.py

```diff
@@ -1279,7 +1279,7 @@ def gen_db_minor_ver_nums(major):
             '/usr/include/db3',
             '/usr/local/include/db3',
             '/opt/sfw/include/db3',
-            # Fink defaults (http://fink.sourceforge.net/)
+            # Fink defaults (https://www.finkproject.org/)
```

Comment:
Old url flakey; if it works, gets forwarded to new page, which was updated yesterday.